src/qml/components/DeveloperOptions.qml

```diff
@@ -19,14 +19,16 @@ ColumnLayout {
     }
     Setting {
         Layout.fillWidth: true
-        header: qsTr("Storage limit")
-        actionItem: OptionSwitch {}
+        header: qsTr("Database cache size")
+        actionItem: ValueInput {
+            description: ("450 MiB")
+        }
     }
     Setting {
         Layout.fillWidth: true
-        header: qsTr("Network")
```

Comment:
4. I was able to verify the non-modifiability by the optionsModel of all the removed settings except this one (”Network”), so could you provide some details on the file responsible for handling the chain switching?

Reply:
there is no file responsible for handling chain switching, that isn't something we can do as of now